test.sh

```diff
@@ -23,9 +23,6 @@ function git_clone() {
 # Remove potential previous runs
 rm -rf src test_program_bin toml-test
 
-# Run go vet
-go vet ./...
```

Comment:
Note that `go test` only runs a limited subset of `vet` tests, as not all of them all considered reliable enough to be included by default (yet). So running `go test` and `go vet` is *not* identical.